test/image/index.html

```diff
@@ -4,6 +4,9 @@
 <!-- this index file gets copied in to the image server docker -->
 <script type="text/javascript" src="../plotly.js/dist/extras/mathjax/MathJax.js?config=TeX-AMS-MML_SVG"></script>
 <script type="text/javascript" src="../plotly.js/build/plotly.js" charset="utf-8"></script>
+<script type="text/javascript">
+    Plotly.setPlotConfig({logging: 0});
+</script>
```

Comment:
@etpinard I added this so `geo_choropleth-text` would render with no console logs (which trip the image server to throw an error). That way the image server keeps the same definition of "error" as it had before. I'm kind of inclined though to either:
1. Downgrade the `Location with id *** does not have a matching topojson feature at this resolution.` message from a `Lib.warn` to a `Lib.log`, or
2. Remove the offending ids from that mock. It's a lot though: ARB, BHR, CEB, CSS, EAP, EAS, ECA, ECS, EMU, EUU, FCS, HIC, HKG, HPC, LAC, LCN, LDC, LIC, LMC, LMY, MEA, MIC, MLT, MNA, NAC, NOC, OEC, OED, OSS, SAS, SGP, SSA, SSF, SST, UMC, WLD

Thoughts?

Reply:
How about both?

> Downgrade the Location with id *** does not have a matching topojson feature at this resolution. message from a Lib.warn to a Lib.log,

Yes, that should certainly be `Lib.log` :ok_hand: 

> Remove the offending ids from that mock. It's a lot though: ARB, BHR, CEB, CSS, EAP, EAS, ECA, ECS, EMU, EUU, FCS, HIC, HKG, HPC, LAC, LCN, LDC, LIC, LMC, LMY, MEA, MIC, MLT, MNA, NAC, NOC, OEC, OED, OSS, SAS, SGP, SSA, SSF, SST, UMC, WLD

I'd vote for removing the offending ids from the mock, but adding a jasmine test spying on `Lib.log` (if not done already). This part here could be done in a separate PR.

Reply:
alright, did both.
- missing location downgraded to `Lib.log` https://github.com/plotly/plotly.js/pull/2195/commits/4b8f9802c3da064a3838c8a7757e170dac952bf3 (we have tests of these messages, modified in that commit)
- removed offending ids anyway https://github.com/plotly/plotly.js/pull/2195/commits/8b9e7d88def8fc3f887a33078cfbd119979c068d